Nautono/RTOS_framework_v1/src/config.h

```diff
@@ -0,0 +1,10 @@
+
+/*
+ * config.h
+ *
+ * Created: 10/29/2023 1:34:52 PM
+ * Author: manav
+ * This files contains all the falgs that modify the behaviour of code.
+ */ 
+
+#define PCB 
```

Comment:
I would add a quick comment about what this does

For example is this for PCB Rev 1 ?